sia-task-core/src/test/java/com/sia/core/helper/StringHelperTest.java

```diff
@@ -0,0 +1,563 @@
+package com.sia.core.helper;
+
+import com.sia.core.helper.StringHelper;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.rules.Timeout;
+
+import java.lang.reflect.Array;
```

Comment:
Is this import (`java.lang.reflect.Array`) used? I guess non-refactored means we don't change anything?

Reply:
Is there a bug for this?

Reply:
https://diffblue.atlassian.net/browse/TG-8393 cover the `java.lang.reflect.Method` case.